dapps/marketplace/webpack.config.js

```diff
@@ -25,7 +25,7 @@ const config = {
   entry: {
     app: './src/index.js'
   },
-  devtool: isProduction ? false : 'cheap-module-source-map',
+  devtool: 'cheap-module-source-map',
```

Comment:
Does this include the source map in the main build? Need to verify it's in a separate file that doesn't get downloaded by clients by default

Reply:
webpack is seriously a strange black box to me.  I just did a test prod build and there's no `.map` files in the build dir.  I also can't seem to find any details about `cheap-module-source-map` [in the webpack docs](https://webpack.js.org/configuration/devtool/) beyond a mention of a setting.  Suggestions?

Reply:
hrm maybe let's just put it back to how it was for now and do this separately...

Reply:
Could be really useful for this... but aight.  Reverted.